tests/ClientTest.php

```diff
@@ -295,12 +295,14 @@ public function testTiming()
         $clientMock->getCategories();
     }
 
-    public function testGetPropertiesReturnsEmptyArrayOnCustomerException()
+    public function testGetPropertiesThrowsCustomerException()
     {
-        $clientMock = $this->getClientMock(array('call'));
-        $clientMock->expects($this->once())->method('call')->will($this->throwException(new CustomerException()));
+        $clientMock = $this->getClientMock(['call']);
+        $clientMock->expects($this->once())->method('call')->willThrowException(new CustomerException());
+
+        $this->expectException(CustomerException::class);
```

Comment:
I personally do, when I expect exceptions, expect them at the top of the test. This way it's clear that there won't be any assertions (usually).
Also please ensure that the exception message is tested (`->expectExceptionMessage()`).